server/athenian/api/openapi/openapi.yaml

```diff
@@ -2742,6 +2742,11 @@ components:
     ReleaseMatchSetting:
       additionalProperties: false
       description: Release matching setting for a specific repository.
+      example:
```

Comment:
I miss the new release [`spec-1.0.37`](https://github.com/athenianco/athenian-api/releases/tag/spec-1.0.37)
Did something happened with the automated process @vmarkovtsev ?

Reply:
There is no tag indeed. I pushed it manually and will investigate why it didn't push automatically tomorrow :+1: